pkg/controller/inference/playground_controller.go

```diff
@@ -188,6 +188,13 @@ func buildWorkloadTemplate(model *coreapi.Model, playground *inferenceapi.Playgr
 		"--served-model-name", modelID,
 		"--port", strconv.Itoa(pkg.DEFAULT_BACKEND_PORT),
 	}
+	if bkd.Name() == inferenceapi.SGLANG {
```

Comment:
I have just refactored this part to make it an interface method, can you change like that:https://github.com/InftyAI/llmaz/pull/47/files#diff-998d843982e36b2a5163d59f67ad4bc159736f103f5a0688bef2ee47b93d8a97R37-R38

Reply:
To avoid the code conflicts, I'll push the refactors first and cc you once I completed. Then you just need to care about the backend implementations.

Reply:
Already rebased on https://github.com/InftyAI/llmaz/pull/48 and one test case has been added too.